packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -11969,6 +11969,45 @@ void main() {
     expect(focusNode.hasFocus, true);
   });
 
+  testWidgets('can change tap up outside behavior by overriding actions', (
+    WidgetTester tester,
+  ) async {
+    bool myIntentWasCalled = false;
+    final CallbackAction<EditableTextTapUpOutsideIntent> overrideAction =
+        CallbackAction<EditableTextTapUpOutsideIntent>(
+          onInvoke: (EditableTextTapUpOutsideIntent intent) {
+            myIntentWasCalled = true;
+            return null;
+          },
+        );
+    final GlobalKey key = GlobalKey();
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Column(
+          children: <Widget>[
+            SizedBox(key: key, width: 200, height: 200),
+            Actions(
+              actions: <Type, Action<Intent>>{EditableTextTapUpOutsideIntent: overrideAction},
+              child: EditableText(
+                autofocus: true,
+                controller: controller,
+                focusNode: focusNode,
+                style: textStyle,
+                cursorColor: Colors.blue,
+                backgroundCursorColor: Colors.grey,
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+    await tester.pump();
+    await tester.tap(find.byKey(key), warnIfMissed: false);
+    await tester.pumpAndSettle();
```

Comment:
Nit: If this works with `pump` instead of `pumpAndSettle`, I would switch to that in order to have less uncertainty in the test.

Reply:
I copied this from the onTapOutside test (the test just above this one), but when running it locally I'm not sure if the pump there is needed at all.